README.md

```diff
@@ -1 +1,51 @@
-# release
\ No newline at end of file
+# Release
+
+This repository will maintain the releases for the aurae project.
```

Comment:
a thought! why is this a separate repository to the `aurae` repository?  wouldn't the releases end up being posted there?

Reply:
Yea my thought is each repo will still have its own releases. But they're all coordinated via this project, and the documentation here provides information on organization wide artifacts

Reply:
This is because we have projects like `ae` that are not part of the `aurae` project, but should be built and released in the same cadence.